libs/SmartSync/src/com/salesforce/androidsdk/smartsync/util/MruSyncDownTarget.java

```diff
@@ -110,6 +110,12 @@ public JSONArray continueFetch(SyncManager syncManager) throws IOException, JSON
         return null;
     }
 
+    @Override
+    public Set<String> getListOfRemoteIds(SyncManager syncManager, Set<String> localIds) {
```

Comment:
Abstract method, so that custom sync down targets can choose how to deal with this.